packages/client3/components/design-system/icon/icon.tsx

```diff
@@ -100,6 +101,7 @@ export const Icon = forwardRef<SVGElement, IconProps>(function Icon(
       aria-hidden="true"
       ref={ref}
       className={clsx(sizeToClassName(size), "inline shrink-0", className)}
+      onClick={onClick}
```

Comment:
Hmm, I thought that every icon would be inside a button and we wouldn't need onClick on the icon, thoughts?

Reply:
I don't really have any concerns to be honest. 

 @v1n33th, thoughts?